cli/permissions.rs

```diff
@@ -328,6 +359,17 @@ impl DenoPermissions {
     permission_prompt(message)
   }
 
+  fn log_perm_access(&self, message: &str) {
+    if log_enabled!(log::Level::Info) {
+      eprintln!(
+        "{}",
+        Style::new()
+          .bold()
+          .paint(format!("ℹ️  Granted {}", message))
```

Comment:
I think we should keep the same emoji for all permission access: ⚠️

Maybe that can be defined as a const at the top of this file?